snap/snapcraft.yaml

```diff
@@ -12,10 +12,17 @@ description:        |
 grade:              stable
 confinement:        strict
 type:               app
+base:               core18
 
 apps:
   krop:
     command:        desktop-launch krop
```

Comment:
You can leave out `desktop-launch`, this is already added by the kde-neon extension.